Assets/HoloToolkit-Examples/Input/Scripts/PopupMenu.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace HoloToolkit.Unity.InputModule.Tests
 {
-    public class PopupMenu : MonoBehaviour, IInputHandler
+    public class PopupMenu : MonoBehaviour
```

Comment:
Why are we removing the input handler?

Reply:
The pop-up menu was only using the input handler to direct modal input in. It additionally had a handler for when the close button is pressed (which essentially did the same thing). Instead, I'm reusing the same event, which means that only the button needs to handle the input events.